Dockerfile

```diff
@@ -482,9 +482,9 @@ RUN prototool lint --protoc-bin-path=/toolchain/bin/protoc --protoc-wkt-path=/to
 
 # The markdownlint target performs linting on Markdown files.
 
-FROM node:8.16.1-alpine AS lint-markdown
-RUN npm i -g markdownlint-cli@0.22.0
-RUN npm i -g textlint@11.6.3
+FROM node:14.5.0-alpine AS lint-markdown
+RUN npm i -g markdownlint-cli@0.23.2
+RUN npm i -g textlint@11.7.6
```

Comment:
For some reason, I was seeing some errors that we never saw in the old setup. Had to bump these to address them.